libraries/Microsoft.Bot.Builder.Dialogs.Declarative/Types/Factory.cs

```diff
@@ -92,10 +90,10 @@ public static void Reset()
             types.Clear();
             names.Clear();
             builders.Clear();
-            RegisterDefaults();
+            RegisterAdaptiveTypes();
         }
 
-        private static void RegisterDefaults()
+        public static void RegisterAdaptiveTypes()
```

Comment:
How do we deal with the configuration dependency? The fact that it is a property means that people may not set it. do we throw when we have to register adaptive types and it is not there?